website/templates/base.mako

```diff
@@ -50,6 +50,27 @@
 
 </head>
 <body data-spy="scroll" data-target=".scrollspy">
+
+    % if private_link_anonymous:
+      ## z-index set to 8 to not overlap with getting started with osf non-logged in user bottom-pop-up
+      ## bottom = 30px to not completely block copyright information
+      <style>
+        #anonymous-mode {
+            position:fixed;
+            bottom:30px; 
+            left:0;
+            z-index: 8;
+            width:100%;
+            text-align:center;
+            background-color: #f0ad4e;
+            padding:.5em;
+        }
+      </style>
+  
+            <div class="" id="anonymous-mode">You are viewing the OSF through an anonymous view-only link. To return to the normal view click <a href="/?view_only=None">here</a></div>
```

Comment:
This message conflicts with the *other* view-only link message, in blue at the top of the screen. The two are visually different without any clear difference in content. We should either rewrite the content to make clear the difference that is visually encoded, or change the visual treatment to make clear that they're the same message—possibly even folding the two messages together.

![screen shot 2016-01-04 at 10 43 45 am](https://cloud.githubusercontent.com/assets/134455/12093284/1822bd8e-b2d0-11e5-82a9-66b2ea61a397.png)